### DISCUSSION TITLE
Tuya TS0601 _TZE204_nqqylykc dimmer

### DISCUSSION THREAD

**Discussion by MichiMess** [Koenkk/zigbee2mqtt#24583] [Devices]
Hi!

I can’t find the same problem on this forum.

I bought some AliExpress tuya ZigBee dimmers but they have other codes. So ZigBee2mqtt doesn’t recognize these.

But the already exist with other codes. See links.

I tried to follow ‘add new device’ but didn’t worked.

Can some help me with this?

Thanks!

Home Assistant with ZigBee2mqtt.

[https://nl.aliexpress.com/item/1005006832261783.html](https://nl.aliexpress.com/item/1005006832261783.html?fbclid=IwZXh0bgNhZW0CMTAAAR3H1tLoqolL-ToqBFHZKAibb58NcU_Vsr5fC4mMSTjlpn0JFy8wPJ1B88U_aem_NNU8LTrdCHC6HSikzvZgGg)

[https://www.zigbee2mqtt.io/devices/ZDMS16-1.html](https://l.facebook.com/l.php?u=https%3A%2F%2Fwww.zigbee2mqtt.io%2Fdevices%2FZDMS16-1.html%3Ffbclid%3DIwZXh0bgNhZW0CMTAAAR31LAYUyU0kHr0w9C_-IX4mIzCDmJjNg43uIYeMBeUM7xBnM_xO6bO99q0_aem_ygPlGZFwUW3m_KHj-uNcOw&h=AT1s6IfbqZodkR6VnfmhAnfg-K-Y21LS5szURzeCAnon87jAYM7jy5BRXjRy9Jv-02UwdvDUIKk2u4ZsslcUHyJWHEfXqXBVIvAjpxl2N5bfJzEvkkJUgeoFJsAqoL0OTA&__tn__=-UK-R&c%5B0%5D=AT0jCvoUmufAHthWGKvjxUNkgzmHVu1P_I0WBJaOEeeREbAChXov1YkXaiZ3d8_zfsnKLZlIS0bUpWJvHI2kXc0d3gTKpijwK7meTGCe9B6iuJhpIT8Gi_WmUsg2Jt8Q3jDTKP3UJ5-ix6Bq6ei47lNeIcHWEJetAs-oOgxnX77oQrmnzpffNHbND7EyUDXwcwDce4P272QGkgaFE6oAarjK_1nYBA)

![465032237_10234754579067224_363250506787000763_n](https://github.com/user-attachments/assets/0343cc3c-da88-47ec-aaba-26ec1a258108)

**Comment by FauthD** [comment]
See this [Solution](https://github.com/Koenkk/zigbee2mqtt/issues/22175).

**Comment by FauthD** [comment]
With the linked solution it somewhat works, but brithness ist not linear, and the connected physical momentary switch behaves strange sometimes.
